Microsoft.Toolkit.Uwp/Commands/DelegateCommand.cs

```diff
@@ -14,16 +14,16 @@
 using System.Diagnostics;
 using System.Windows.Input;
 
-namespace Microsoft.Toolkit.Uwp.SampleApp.Common
+namespace Microsoft.Toolkit.Uwp.Commands
```

Comment:
Would like to get other's thoughts on this. I know other toolkits have it under a Commands namespace.